app/helpers/view_components/student/concepts_map.rb

```diff
@@ -0,0 +1,15 @@
+module ViewComponents
+  module Student
+    class ConceptsMap < ViewComponent
+      def initialize(_data)
+        @data
+      end
```

Comment:
You've not actually set `@data` here. Regardless, you can use `initialize_with` from Mandate. It's already extended in `ViewComponent`

```suggestion
     initialize_with :data
```